src/gui/mainwindow.cpp

```diff
@@ -187,13 +189,23 @@ MainWindow::MainWindow(IGUIApplication *app, WindowState initialState)
     hSplitter->setChildrenCollapsible(false);
     hSplitter->setFrameShape(QFrame::NoFrame);
 
-    // Name filter
+    // Torrent filter
     m_searchFilter = new LineEdit(this);
-    m_searchFilter->setPlaceholderText(tr("Filter torrent names..."));
+    m_searchFilter->setPlaceholderText(tr("Filter torrents..."));
     m_searchFilter->setFixedWidth(200);
     m_searchFilter->setContextMenuPolicy(Qt::CustomContextMenu);
     connect(m_searchFilter, &QWidget::customContextMenuRequested, this, &MainWindow::showFilterContextMenu);
     m_searchFilterAction = m_ui->toolBar->insertWidget(m_ui->actionLock, m_searchFilter);
+    // Filter by type
+    QWidget *filterByWidget = new QWidget(this);
```

Comment:
This name is confusing. I would use something like `filterColumnWidget` instead.

Reply:
And rename all the related widgets/controls accordingly: `filterColumnLabel`, `filterColumnLayout`, `m_filterColumnComboBox` etc.

Reply:
We prefer to avoid duplicating type name in statements like this one.
```suggestion
    auto *filterByWidget = new QWidget(this);
```